### ISSUE TITLE
search

### ISSUE THREAD

**Issue by itman0101** [xDae/vue-fireMovie#3]
the search part in not working

**Comment by xDae** [comment]
@itman0101 yes, the code is old (Vue v1), I would like to update and refactor the code with last Vue versions (need time)
